GliaWidgets/Component/Header/Header.swift

```diff
@@ -21,10 +21,13 @@ class Header: UIView {
             }
         }
     }
-
+    var backButton: HeaderButton
+    var closeButton: HeaderButton
+    var endButton: ActionButton
+    var endScreenSharingButton: HeaderButton
```

Comment:
I'm pretty sure it was called `endScreenShareButton` previously. Not a big deal, just a continuity/consistency issue.

However, a quick search of "screensharing" reveals a bunch of other places in the code where such a naming scheme is used. Even in the SalemoveSDK events 😄  But let's at least call the button the same way as it was called previously.